specs/engine.spec/src/main/scripts/io/aklivity/zilla/specs/engine/schema/metrics/engine.schema.patch.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "op": "add",
+        "path": "/$defs/telemetry/metrics/items/enum/-",
+        "value": "engine.worker.utilization"
+    }
+]
```

Comment:
I think we shouldn't need this to be configured, it should be always on, even for empty config, agree?